modules/auxiliary/gather/ip_geolocate.rb

```diff
@@ -0,0 +1,103 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
```

Comment:
No, I mean it's already being loaded earlier in Framework, so you don't have to `require` it again. This is one of the biggest sources of start time!